management-api-for-apache-cassandra.advisories.yaml

```diff
@@ -11,7 +11,7 @@ advisories:
       - timestamp: 2024-01-14T18:15:03Z
         type: pending-upstream-fix
         data:
-          note: "To fix the CVE, we have to upgrade 'swagger-jaxrs2' to '2.2.0' or later but this fix will require some code change since the upgrade cause to build fail due to compilation errors like: 'src/main/java/com/datastax/mgmtapi/resources/LifecycleResources.java:[425,28] cannot access com.fasterxml.jackson.core.exc.StreamWriteException'"
+          note: 'To fix the CVE, we have to upgrade ''swagger-jaxrs2'' to ''2.2.0'' or later but this fix will require some code change since the upgrade cause to build fail due to compilation errors like: ''src/main/java/com/datastax/mgmtapi/resources/LifecycleResources.java:[425,28] cannot access com.fasterxml.jackson.core.exc.StreamWriteException'''
```

Comment:
Yes, these look odd. But I'd prefer to keep these changes for now. I'd rather we have a consistent way of handling string quotes in the code throughout, and we can always change our approach in the future, while maintaining consistency at the same time.